src/main/java/org/gradle/profiler/studio/StudioGradleClient.java

```diff
@@ -103,6 +139,7 @@ public void close() {
                     System.out.println("* Android Studio stopped.");
                 });
             }
+            executor.shutdown();
```

Comment:
Can we move the `executor.shutdown()` at the beginning of the `try` block?

Reply:
We can do that. Moved.